lib/core/src/Cardano/Wallet/Primitive/Types/RewardAccount.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
+-- License: Apache-2.0
+--
+-- This module provides the 'RewardAccount' data type, used for staking
+-- purposes.
+--
+module Cardano.Wallet.Primitive.Types.RewardAccount
+    ( RewardAccount (..)
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Primitive.Types.Hash
+    ( Hash (..) )
+import Control.DeepSeq
+    ( NFData (..) )
+import Data.ByteString
+    ( ByteString )
+import Data.Text.Class
+    ( FromText (..), ToText (..) )
+import Fmt
+    ( Buildable (..) )
+import GHC.Generics
+    ( Generic )
+
+-- | Also known as a staking key, a reward account is used in group-type
```

Comment:
I don't think "staking key" is correct - this term does not appear in the delegation design spec.

Reply:
> I don't think "staking key" is correct - this term does not appear in the delegation design spec.

Removed in c21aabab7e0ee2a318e056a5298800f6c99ca795.